dev/partkey.md

```diff
@@ -172,3 +172,41 @@ wrapped in a $\Credential$ struct, containing the following fields:
 
 And this verified credential is wrapped in a $\Vote$ struct with _Raw Vote_ 
 ($\mathrm{R}$), _Verified Credential_ ($\Credential$), and _Signature_ ($\Sig$).
+
+
+## Algorand State Proof Keys 
+### Algorand's Committable Ephemeral Keys Scheme - Merkle Keystore
+
+Algorand archives [forward security](https://en.wikipedia.org/wiki/Forward_secrecy) using the Merkle Keystore scheme. This scheme consists of designate an ephemeral key for each round in which it will be used. The scheme uses a Merkle tree to generate a commitment on those keys. 
+The private key must be deleted be the done in order the completely archives forward secrecy.
+
+The ephemeral key can be generated using one of Algorand's supported digital signature algorithms.
```

Comment:
Isn't the state proof using FALCON in all cases?
Or do you allow to be used also with ED25519?
Would both proofs be generated?